src/pushNotifications/TokenHandling.ts

```diff
@@ -0,0 +1,75 @@
+import { Platform } from 'react-native';
+
+import { secrets } from '../config';
+import * as SecureStore from 'expo-secure-store';
+import * as appJson from '../../app.json';
```

Comment:
please order imports in the following way:

```
import { Platform } from 'react-native';
import * as SecureStore from 'expo-secure-store';

import { secrets } from '../config';
import * as appJson from '../../app.json';
```

as we import from libraries first and local stuff second, divided by a line break, all over the codebase to maintain fast readability.

Reply:
done in 5c4968a26c167de3f0343c155960812f53460942